### DISCUSSION TITLE
Cannot link an issue to a project

### DISCUSSION THREAD

**Discussion by eirys** [community/community#45412] [Mobile]
I'm new to GitHub (mobile and laptop) but it seems like I cannot associate an issue to a project. Is it just me who has bad settings somehow or is it a bug ?

**Comment by jge162** [comment]
It is possible that you do not have the necessary permissions to associate an issue with a project in the GitHub repository. In GitHub, permissions are set at the repository level, so you will need to check the repository's settings to see what level of access you have.

Go to settings, collaborators and see your permissions you have.

**Comment by eirys** [comment]
Thanks for the reply. Actually I can't link issues to my own projects/repositories.
I checked my other existing issues (that I linked to my project while on GitHub desktop version), they are marked as unassociated as well.
